docs/DeveloperGuide.md

```diff
@@ -321,6 +324,103 @@ The following sequence diagram shows how the `login` command works:
 
 <puml src="diagrams/UserLogoutSequenceDiagram.puml" alt="UserLogoutSequenceDiagram" />
 
+//Can state somewhere that all commands need to be logged in before can use
```

Comment:
Currently appears in the markdown file. Perhaps can include the check of login status within the activity/sequence diagrams and description?

Reply:
ok lets discuss with the rest and kiv